src/loaders/features/features.js

```diff
@@ -0,0 +1,37 @@
+export const FEATURE_NAMES = {
+    ajax: "ajax",
+    jserrors: "jserrors",
+    metrics: "metrics",
+    pageAction: "page_action",
+    pageViewEvent: "page_view_event",
+    pageViewTiming: "page_view_timing",
+    sessionTrace: "session_trace",
+    spa: "spa"
+}
+
+export const asyncFeatures = [
+    FEATURE_NAMES.metrics
+]
+
+export const syncFeatures = [
```

Comment:
Probably already covered in a previous comment on commenting & docs, but doc here would be nice, as this is confusing to new folks who would think that these features actually operate in sync (block UI thread for long periods, don't use `setTimeout` or async functions, etc.)

Reply:
https://issues.newrelic.com/browse/NEWRELIC-6479